corehq/apps/locations/bulk_management.py

```diff
@@ -315,14 +315,13 @@ def validate_and_parse_data_from_excel(self):
         actual = set(type_sheet_reader.headers)
         expected = set(LOCATION_TYPE_SHEET_HEADERS.values())
         if actual != expected:
-            message = ugettext_lazy("'types' sheet should contain exactly '{expected}' as the sheet headers. "
-                                    "'{missing}' are missing")
-            if actual - expected:
-                message = format_lazy('{}{}', message, ugettext_lazy(" '{extra}' are not recognized"))
+            missing = ", ".join(expected - actual)
```

Comment:
nit: might want to rename these to missing_headers to avoid confusion with the variables in translation.

Reply:
going to ignore this just because this PR became a bit bigger than i expected, and i'd just like to get it merged

Reply:
Sure @orangejenny